lib/webserver/esp-fs-webserver.cpp

```diff
@@ -183,6 +183,13 @@ IPAddress FSWebServer::startWiFi(uint32_t timeout, const char *apSSID, const cha
     _ssid = reinterpret_cast<const char *>(conf.sta.ssid);
     _pass = reinterpret_cast<const char *>(conf.sta.password);
 
+    char * my_ssid;
+    my_ssid = new char[32+1];
+    strncpy(my_ssid, _ssid, 32);
+    my_ssid[32] = '\0';
+    if (strlen(my_ssid) < strlen(_ssid))
```

Comment:
In my opionion, the condition is unnecessary because my_ssid is always either equal to or shorter than _ssid after copying, making the condition unnecessary.

Reply:
I think you are right. As I'm not very good at C++, this was my safety check to only use my modified ssid, when really needed. That's why the check for shorter (not shorter equal).

I can remove the check if you think it is better.

Reply:
No worries, I think It doesn't matter.